packages/angular_devkit/build_angular/src/builders/extract-i18n/ivy-extract-loader.ts

```diff
@@ -21,6 +21,11 @@ export default function localizeExtractLoader(
   content: string,
   map: LoaderSourceMap,
 ) {
+  // This loader is not cacheable due to message extraction works.
+  // Extracted messages are not part of webpack pipeline and hence they cannot be retrieved from cache.
+  // TODO: We should investigate in the future on making this deterministic and more cacheable.
```

Comment:
Should we file an issue or project about this? I'm always hesitant about `TODO`'s in the code which tend to be forgotten about. Issues are more easily triaged, assignable, and actionable IMHO.

Reply:
Agreeded